<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="id_line">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="id_sign_lable">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>검색할 아이디를 입력해주세요.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_btn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancle_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QLabel" name="sign_lable">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="date_sign_lable">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>저장한 날짜를 입력해주세요.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="date_line">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="num_sign_lable">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>조회 할 번호를 입력해주세요.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="num_line">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
